code/modules/power/power.dm

```diff
@@ -477,7 +477,7 @@
 //siemens_coeff - layman's terms, conductivity
 //dist_check - set to only shock mobs within 1 of source (vendors, airlocks, etc.)
 //No animations will be performed by this proc.
-/proc/electrocute_mob(mob/living/carbon/victim, power_source, obj/source, siemens_coeff = 1, dist_check = FALSE)
+/proc/electrocute_mob(mob/living/carbon/victim, power_source, obj/source, siemens_coeff = 1, dist_check = FALSE, max_damage = 200)
```

Comment:
this argument may be misleading
   - there is already a cap on elecrocution damage, that being 195 +- 5, calculated in get_electrocute_damage. 
   - this means consumers may read this, think "for my situation, i want a higher cap than the default", and then get confused when it doesn't apply. 

this means the "correct" way to accomplish what you're accomplishing would be to make a "power cap", not a damage cap, where you pass some value to it saying "do not consume more than this many joules".

I think for 80 damage, you'd want like... `1500 KILO JOULES` / `1.5 MEGA JOULES`?

Reply:
good point, i'll make the adjustment and test it. I'll also reword the PR as well